meta-sturgeon/recipes-android/brcm-patchram-plus/brcm-patchram-plus_git.bbappend

```diff
@@ -1,3 +1,7 @@
 FILESEXTRAPATHS:prepend:sturgeon := "${THISDIR}/brcm-patchram-plus:"
 SRC_URI:append:sturgeon = " file://patchram.service "
 CFLAGS:append:sturgeon = " -DLPM_STURGEON"
+
+do_install:append:sturgeon() {
```

Comment:
Could you add a comment to explain that this disables the systemd service ? Or be a bit more specific and rm the actual service file